releasenotes/notes/no-linux-32bit-cd10b5e02e83674b.yaml

```
@@ -0,0 +1,5 @@
+---
+deprecations:
```

```suggestion
upgrade:
```

?


```suggestion
other:
```

?


I read "deprecation" notes more as "hey, this still works, but is going to go away at some point", this change we are basically removing support for 32-bit linux?

or are we just trying to "soft support" today, but since we don't test or build for it there is no promise that things work?

---

didn't think that thoroughly. By dropping linux 32-bit, I'd mean that we don't support it at all, so it's certainly not "hey, this sill works" nor "soft support". It's at your own risk if you'd like to use it on 32-bit linux